wandb/integration/sb3/sb3.py

```diff
@@ -93,7 +93,9 @@ def __init__(
             ), "to use the `model_save_freq` you have to set the `model_save_path` parameter"
 
     def _init_callback(self) -> None:
-        d = {"algo": type(self.model).__name__}
+        d = {}
+        if "algo" not in d:
+            d["algo"] = type(self.model).__name__
```

Comment:
For my own edification, can you explain why this is needed?

Reply:
In case the user wants to override the model’s name like “PPO-LSTM”. When grouping the algorithms in the dashboard it could be quite convenient. I don’t have a strong push for adding this though.